Source/astcenccli_platform_dependents.cpp

```diff
@@ -165,6 +187,18 @@ double get_time()
 	return static_cast<double>(tv.tv_sec) + static_cast<double>(tv.tv_usec) * 1.0e-6;
 }
 
+/* See header for documentation */
+void set_thread_name(
+	const char* name
+) {
+	// No standard mechanism, so be defensive here
```

Comment:
For linux at least you can use https://man7.org/linux/man-pages/man2/PR_SET_NAME.2const.html

    prctl(PR_SET_NAME, name, 0, 0, 0);

but it all amounts to the same thing i suppose